src/transaction_utils.cpp

```diff
@@ -367,6 +368,17 @@ namespace sdk {
             net_params, result, tx, address, satoshi.value(), addressee.value("asset_tag", std::string{}));
     }
 
+    void set_tx_output_value(const network_parameters& net_params, wally_tx_ptr& tx, uint32_t index,
```

Comment:
maybe `satoshi` instead of `value`? I feel like value is overloaded in a lot of places.

Reply:
Is now `satoshi`